### PR TITLE
Add analytics

### PR DESCRIPTION
**Pull request by saundefined**
RFC: https://wiki.php.net/rfc/phpnet-analytics

/cc @pronskiy @crell

### CODE REVIEW

File: include/header.inc [hunk 1]
```diff
@@ -91,6 +91,23 @@ if (!isset($config["languages"])) {
...
+<!-- Matomo -->
+<script>
+    var _paq = window._paq = window._paq || [];
+    /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+    _paq.push(["setDoNotTrack", true]);
+    _paq.push(["disableCookies"]);
+    _paq.push(['trackPageView']);
+    _paq.push(['enableLinkTracking']);
+    (function() {
+        var u="//analytics.php.net/";
```
Comment (saundefined) [inline include/header.inc:103]:
```suggestion
        var u="https://analytics.php.net/";
```

**Review by sy-records — APPROVED** [review]

File: include/header.inc [hunk 2]
```diff
@@ -91,6 +91,23 @@ if (!isset($config["languages"])) {
 
 <?php if (isset($config['meta_tags'])) { echo $config['meta_tags']; } ?>
 
+<!-- Matomo -->
```
Comment (sy-records) [inline include/header.inc:94]: Should only `php.net` be allowed? exclude `*.preview.thephp.foundation`

Comment (saundefined) [reply include/header.inc:94]: We have enabled "Only track visits and actions when the action URL starts with https://www.php.net" in Matomo,

but I guess we can add this one for double-check 🤔

Comment (saundefined) [reply include/header.inc:95]: @sy-records updated

**Review by sy-records — APPROVED** [review]

**Comment by leonardolara** [comment]
That's just great! I was wondering if there were any page access statistics for php.net.
It will be a great source of information for translators.
Any idea on how the info will be made available?

**Comment by leonardolara** [comment]
I would like to see, for example, the page access counts (in English and Portuguese translation) from a geographic location (specially those inside Portuguese-speaking countries - Brazil, Portugal, Angola, Mozambique, etc). These stats will help me to prioritize the pt_BR translation efforts.

**Comment by leonardolara** [comment]
> I would like to see, for example, the page access counts (in English and Portuguese translation) from a geographic location (specially those inside Portuguese-speaking countries - Brazil, Portugal, Angola, Mozambique, etc). These stats will help me to prioritize the pt_BR translation efforts.

@pronskiy, have you gathered enough data for these statistics?

**Pull request merged**